main/utilities/OI_Utils.java

```diff
@@ -20,28 +20,19 @@ public static Supplier<Double> getAxisSupplier(Control axis) {
     return () -> axis.getCurve().calculate(axis.getController().getRawAxis(axis.getId()));
   }
 
-  public static JoystickButton getButton(Control button) {
-    if (button.getType() != Control.ControlType.BUTTON) {
-      DriverStation.reportError(button.getAction() + " is not a button", true);
+  public static Trigger getButton(Control button) {
+    switch (button.getType()) {
+      case BUTTON:
+        return new JoystickButton(button.getController(), button.getId());
+      case POVBUTTON:
+        return new POVButton(button.getController(), button.getId());
+      case TRIGGER:
+        return new Trigger(
+            () -> button.getController().getRawAxis(button.getId()) > button.getThreshold());
+      default:
+        DriverStation.reportError(button.getAction() + " is not a valid input", true);
+        return new Trigger(() -> false);
```

Comment:
We should make this a runtime error, so the code crashes. Prevents these issues from propogating